config.py

```diff
@@ -34,10 +34,13 @@ def getenv(key, default: Optional[str] = None):
     MOCK_CANVAS = getenv('MOCK_CANVAS', 'false').lower() == 'true'
     SEND_EMAIL = getenv('SEND_EMAIL', 'off').lower()
 
-    # Email setup. Domain environment is for link in email.
-    SENDGRID_API_KEY = getenv('SENDGRID_API_KEY', "placeholder")
+    # Email setup.
+    EMAIL_SERVER = getenv('EMAIL_SERVER', "unset")
+    EMAIL_PORT = getenv('EMAIL_PORT', "unset")
+    EMAIL_USERNAME = getenv('EMAIL_USERNAME', "unset")
+    EMAIL_PASSWORD = getenv('EMAIL_PASSWORD', "unset")
 
-    PHOTO_DIRECTORY = getenv('PHOTO_DIRECTORY', "placeholder")
+    PHOTO_DIRECTORY = getenv('PHOTO_DIRECTORY', "unset")
```

Comment:
I wonder if it makes sense to use `None` for default values?

Reply:
If we don't specify a default value it is already `None`.

The getenv helper I wrote will raise exception when the variable is `None`.

```py
class ConfigBase(object):

    @staticmethod
    def getenv(key, default: Optional[str] = None):
        val = os.getenv(key, default)
        if val is None:
            raise EnvironmentalVariableMissingError(key)
        return val
```

Providing a non-`None` default value is a way to make this var optional.

But they are not really optional - if you want the email to be actually sent then you have to provide it.

But in DEV env then you don't want email to be sent. They don't need to be there.

Erhh sounds a bit messy. Maybe for the email env vars I should just put them into `DevelopmentConfig`